### DISCUSSION TITLE
Enhancing the docs for SelectPicker

### DISCUSSION THREAD

**Discussion by JoueBien** [rsuite/rsuite#3627] [RFCs]
With this Discussion I'd like to see if there is apatite for me to improve the docs for SelectPicker? 
Primarily around the use of the container prop & how options are stuck to the page scroll by default. The lack of explanation for these features has led to a few bug reports  https://github.com/rsuite/rsuite/issues?q=SelectPicker+scroll likely because the latter is not a behaviour everyone expects.

The main issues I'd like to resolve in the docs are:
-  There isn't an example of using SelectPicker's container prop for setting the rendering location of the options. There is sort of an example under the "Container and prevent overflow [#](https://rsuitejs.com/components/select-picker/#container-and-prevent-overflow)" heading but it really only provides an example of using PreventOverflowContainer.
- The description for the container prop is misleading. Currently it is "Sets the rendering container" when in actuality it should say something along the lines of "allows you to set the rendering location of the option's portal".
- For those that want options not stuck to the scroll - how to do that isn't provided as an example. You'd have to read two closed bug reports to figure out how. With the how being: the container element must have a position of relative & the container element's ref must be passed in to SelectPicker.

Possibly with a follow up PR to look at:
- The size of the page for SelectPicker is verry long it would be easier to read & consume if it was split up into multiple pages, better organised & examples had a short preamble. This is kind of ones of those opinionated things, so I'm willing to leave it alone if it's too much of a hassle.

**Comment by simonguo** [comment]
Hi @JoueBien , thank you for your feedback. There are indeed some areas in the document that need to be improved. If you are interested in this, you can submit a PR to help us improve the document.
